### PR TITLE
Feat/article with video label

### PR DESCRIPTION
**Pull request by nikhedonia**

### CODE REVIEW

File: packages/article/get-lead-asset.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export default function getLeadAsset({ leadAsset: asset }) {
+  if (!asset) return { isVideo: false, leadAsset: null };
+
+  const isVideo = !!asset.posterImage;
```
Comment (aronshamash) [inline packages/article/get-lead-asset.js]: Is this a guarantee? will it always be populated?

File: packages/video-label/style/shared.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  title: {
+    fontFamily: fonts.supporting,
+    fontSize: 12,
+    lineHeight: 12,
+    fontWeight: "400",
+    letterSpacing: 1.2,
+    padding: 0,
+    margin: 0,
+    position: "relative",
+    top: 2
```
Comment (aronshamash) [inline packages/video-label/style/shared.js]: Can we remove this and increase the `lineHeight`?

File: packages/article/get-lead-asset.js (hunk 1 above)
Comment (nikhedonia) [reply packages/article/get-lead-asset.js]: Yes, It comes directly from the graphql-endpoint.

File: packages/video-label/style/shared.js (hunk 2 above)
Comment (nikhedonia) [reply packages/video-label/style/shared.js]: Is it desired to remove the lineHeight in `article-label` also?

Comment (nikhedonia) [reply packages/video-label/style/shared.js]: I'm unable to center the text vertically by doing so.

File: packages/article/get-lead-asset.js (hunk 1 above)
Comment (craigbilner) [reply packages/article/get-lead-asset.js]: this isn't how you should determine a video though, you have a type system?

File: packages/article/article-header/article-header-label.js [hunk 3]
```diff
@@ -1,24 +1,28 @@
 import React from "react";
 import PropTypes from "prop-types";
 import ArticleLabel from "@times-components/article-label";
+import VideoLabel from "@times-components/video-label";
```
Comment (craigbilner) [inline packages/article/article-header/article-header-label.js]: the native and web versions are basically the same, would be tempted to create a base version with render prop?

File: packages/article/fixtures/article-with-video-asset.json [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+              "attributes": {
+                "value":
+                  "Tony Hartley, who lives 50 yards away from the site, said he had found a man buried beneath the rubble. “Me and a friend lifted up a steel girder with about five other blokes and removed a man"
+              },
+              "children": []
+            }
+          ]
+        }
+      ],
+      "leadAsset": {
```
Comment (nehasri89) [inline packages/article/fixtures/article-with-video-asset.json]: A video fixture should be something along the lines of http://cps-api-tnl-prod.elb.tnl-prod.ntch.co.uk/v0.8/document/sir-ken-dodd-dies-aged-90-two-days-after-marriage-32h7t6p2s

Comment (nehasri89) [reply packages/article/fixtures/article-with-video-asset.json]: We have a story in the backlog for video integration, just wondering if there is any specific reason / requirement for doing just the video label or is the video integration already done if I may have missed it

Comment (nehasri89) [reply packages/article/fixtures/article-with-video-asset.json]: 👍

File: packages/article/get-lead-asset.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+export default function getLeadAsset({ type, leadAsset: asset }) {
+  if (!asset) return { isVideo: false, leadAsset: null };
+
+  const isVideo = asset.type === 'Video';
```
Comment (aronshamash) [inline packages/article/get-lead-asset.js]: better

File: packages/article/article-header/article-header-label.js (hunk 3 above)
Comment (aronshamash) [reply packages/article/article-header/article-header-label.js]: This makes sense @nikhedonia

Comment (nikhedonia) [reply packages/article/article-header/article-header-label.js]: Let's tackle this in a seperate PR

Comment (nikhedonia) [reply packages/article/article-header/article-header-label.js]: create issue #723 to document this as a todo.

**Pull request merged**